Cargo.toml

```diff
@@ -1,54 +1,54 @@
 [workspace]
-resolver = "2"
-members = [
```

Comment:
Same here.

Reply:
I think this particular case can we worked around by putting another newline in between `resolver` and the subsequent fields?